examples/test/VisualizationByIdentifier_test.js

```diff
@@ -93,3 +93,7 @@ test("Treemap should render", async t => {
 test("Heatmap should render", async t => {
     await checkRenderedChart(".s-visualization-heatmap", t);
 });
+
+test("Geo Pushpin chart should render", async t => {
```

Comment:
I prefer `GeoPushpinChart should render`.
What do you think?

Reply:
fixed